functionary/ui/views/function.py

```diff
@@ -65,18 +65,22 @@ def get_context_data(self, **kwargs):
 @require_POST
 @login_required
 def execute(request: HttpRequest) -> HttpResponse:
```

Comment:
NOTE: I left the execute function here, even though it is no longer strictly for functions. This was to make it easier to review since you can actually see the changes in the diff. We can move it to a more appropriate home later (probably the task view?) if it really bothers us that it lives here.